js/mviewer.js

```diff
@@ -1843,6 +1901,8 @@ mviewer = (function () {
                 _initDisplayMode();
                 _initDataList();
                 _initVectorOverlay();
+                _initSelectOverlay();
+                _initSubSelectOverlay();
```

Comment:
Dont understand why you create 2 layers (subh and select) ?

You could directly manage style inside only one vector layer to separate sub and selected.

You already have the code to detect if feature is sub or selected feature when you remove select overlay from subSelectedOverlay.

Reply:
I agree and played around with the styling alternative, but this makes it very hard to keep the "sub-selected" feature on top of the others (z-index). Unselecting other features also gets more complicated. Using two layers makes both very simple even though, the concept might not be the most elegant.

Reply:
>  Unselecting other features also gets more complicated

Could you maybe create some fill for selected and some stroke for others to pass z-index issue ?

Reply:
Yes I tried with different fills and strokes, but this did not resolve the issue to often end up with the sub selected feature to be overlayed by other selected features